### DISCUSSION TITLE
is the editable reblogs not working for anyone else?

### DISCUSSION THREAD

**Discussion by itspeteski** [AprilSylph/XKit-Rewritten#1282] [Help Desk] [answered]
is the editable reblogs not working for anyone else? noticed it this past week, looks like there was a dash update? and now I cannot reblog (my own stuff) and edit it. 

If any one nows how to remove the **janky looking indented reblog format** displayed (not on dash but on your site) - that's my main reason for editing reblogs (my own : ) 

Pete

**Comment by AprilSylph** [comment, marked as answer]
Editable Reblogs is part of New XKit, only works in the legacy editor (which has been removed), and is no longer maintained.

See https://github.com/AprilSylph/XKit-Rewritten/wiki/Features#trim-reblogs for its replacement.

**Reply by itspeteski** [reply]
thanks April! There is a work around (re: editable reblogs from desktop dash)  if anyone is interested, let me know :  )

**Reply by KonstancjaMG** [reply]
@itspeteski please, let me know cause Trims doesn't work for me.

**Reply by itspeteski** [reply]
this only works as far as I can tell with reblogging my own posts

from the desktop if you have "share to tumblr" bookmarked 
if you don't here is the bookmark/script  - copy into the address bar - bookmark 
(it would be better to create a bookmark, with a random address then go to edit that bookmark and add the code as the address) 
`javascript:var d=document,w=window,e=w.getSelection,k=d.getSelection,x=d.selection,s=(e?e():(k)?k():(x?x.createRange().text:0)),f=%27http://www.tumblr.com/share%27,l=d.location,e=encodeURIComponent,p=%27?v=3&u=%27+e(l.href) +%27&t=%27+e(d.title) +%27&s=%27+e(s),u=f+p;try{if(!/^(.*\.)?tumblr[^.]*$/.test(l.host))throw(0);tstbklt();}catch(z){a =function(){if(!w.open(u,%27t%27,%27toolbar=0,resizable=0,status=1,width=450,height=430%27))l.href=u;};if(/Firefox/.test(navigator.userAgent))setTimeout(a,0);else a();}void(0)`

so, go to permanent link of desired post to reblog

"share to tumblr" (using above)

post (click reblog)

go to post on dash

edit

bingo, there is your editable reblog 

let me know how you make out

**Reply by AprilSylph** [reply]
@KonstancjaMG if Trim Reblogs isn't working for you, please open an issue detailing what you tried and what went wrong.

It may also help to read [the user guide](https://github.com/AprilSylph/XKit-Rewritten/wiki/Features#trim-reblogs) for Trim Reblogs, since it works significantly differently from Editable Reblogs.

**Reply by KonstancjaMG** [reply]
@AprilSylph All works, I'm just dumb. Skipped over the fact it's a completely new Xkit extension, not just an update to the older version. You're the best!

@itspeteski Thanks for taking the time. It indeed only works for own posts.
